scripts/prepare-package-for-release.js

```diff
@@ -81,9 +81,20 @@ if (
   exit(1);
 }
 
+// Release builds should commit the version bumps, and create tags.
+echo('Updating RNTester react-native dependency...');
+if (
+  exec(`source scripts/update_rn_tester.sh && update_package_json ${version}`)
```

Comment:
The alternative to `jq` would be using Node.js APIs, instead of handing off to shell scripts in this workflow — but I'm happy to come back to this later.

Reply:
As in, remove the shell script altogether and rewrite in JS correct? I didn't do that so as to limit risk but I can keep that in mind :)

Reply:
Yup. Not important for now, but I'm likely to come back and refactor this next :D